create-postgres/action.yml

```diff
@@ -0,0 +1,38 @@
+name: "Create Postgres"
+description: "Create Postgres"
+inputs:
+  api-token:
+    description: "API Token."
+    required: true
+    default: ""
+  cluster-id:
+    description: "Cluster id to attach postgres to"
+    required: true
+    default: ""
+  version:
+    description: "Postgres version to provision."
+    required: false
+  instance-type:
+    description: "Instance type to provision"
+    required: false
+  disk:
+    description: "Disk size in GiB"
```

Comment:
The cli sets the default to be 200 GiB. Do we need to do the same or will the api handle that?

Reply:
Yes, if empty, the api should handle it.